src/pages/Admin/Room/Edit/RoomEdit.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { useEffect, useState } from "react";
-import { Form, Modal } from "react-bootstrap";
-import { useForm } from "react-hook-form";
 import ImageInput from "components/atoms/ImageInput";
-
-import * as S from "./RoomEdit.styles";
-import { CustomInputsType, RoomTemplate, ROOM_TEMPLATES } from "settings";
-import { Button } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { Form, Modal, Button } from "react-bootstrap";
+import { RoomTemplate, ROOM_TEMPLATES } from "settings";
 import { EditRoomProps } from "./RoomEdit.types";
 
-const EditRoom: React.FC<EditRoomProps> = ({
+import "./RoomEdit.scss";
+
+export const EditRoom: React.FC<EditRoomProps> = ({
```

Comment:
We should align the filename and the main component's name here. These should both be one of `RoomEdit` or `EditRoom` (not both)

Since this seems to be a Modal, it may also be beneficial to suffix this with `Modal`, making it one of `RoomEditModal` or `EditRoomModal`.

I feel like @denisdimitrov made some naming changes related to this (or a similar pattern) recently. I think he started with a noun from memory.. so making this `RoomEditModal` would probably be the preference.

Reply:
Changed everything to RoomEditModal. Question: should I rename the ```EditRoomProps``` ```RoomEditModalProps``` or something else because that sounds a little too long?

Reply:
I would keep at as `RoomEditModalProps`, as that's the pattern we're following. While it is kind of long, I don't think it matters that much. In all likelihood it's only ever going to get used in 1 place within this file anyway.